### DISCUSSION TITLE
Updates?

### DISCUSSION THREAD

**Discussion by HandsomeJussi** [builtbybel/FlyOOBE#36] [General]
Is there a tool or can Flyby11 be used for updates? 

Apparently I am missing important safety and quality updates but at the same time my system is up to date....

Used instructions and tried to update via ISO but that checks the system requirements again -.-'
![Näyttökuva 2024-12-11 101919](https://github.com/user-attachments/assets/abcc5ff7-c23e-4627-85a0-625b232abb14)

If there is no solution please make a tool for this too :(

**Comment by otaktn** [comment]
Měl jsem stejný problém. Přeinstaloval jsem Win11 pomocí FlyBy11 ver. 1.2 a problém zmizel.

I had the same problem. I reinstalled Win11 using FlyBy11 version 1.2, and the problem disappeared
